modules/renderer/background_source.js

```diff
@@ -222,18 +222,18 @@ rendererBackgroundSource.Bing = function(data, dispatch) {
     var providers = [];
 
     d3_json(url, function(err, json) {
-        providers = json.resourceSets[0].resources[0].imageryProviders.map(function(provider) {
-            return {
-                attribution: provider.attribution,
-                areas: provider.coverageAreas.map(function(area) {
-                    return {
-                        zoom: [area.zoomMin, area.zoomMax],
-                        extent: geoExtent([area.bbox[1], area.bbox[0]], [area.bbox[3], area.bbox[2]])
-                    };
-                })
-            };
-        });
-        dispatch.call('change');
+        // providers = json.resourceSets[0].resources[0].imageryProviders.map(function(provider) {
+        //     return {
+        //         attribution: provider.attribution,
+        //         areas: provider.coverageAreas.map(function(area) {
+        //             return {
+        //                 zoom: [area.zoomMin, area.zoomMax],
+        //                 extent: geoExtent([area.bbox[1], area.bbox[0]], [area.bbox[3], area.bbox[2]])
+        //             };
+        //         })
+        //     };
+        // });
+        // dispatch.call('change');
```

Comment:
why this commenting out?

Reply:
something was wrong with retrieving the link (checked on others machines as well) and it seemed to be causing test failures so i commented it out, but looks like it has been fixed so i readded this